src/webots/nodes/WbInertialUnit.cpp

```diff
@@ -126,8 +126,11 @@ void WbInertialUnit::computeValue() {
   // get north and -gravity in global coordinate systems
   const WbWorldInfo *const wi = WbWorld::instance()->worldInfo();
   const WbVector3 &north = wi->northDirection().normalized();
-  const WbVector3 &minusGravity = -wi->gravity().normalized();
+  WbVector3 minusGravity = -wi->gravity().normalized();
 
+  if (minusGravity.isNan()) {
+    minusGravity = WbVector3(0.0, 1.0, 0.0);
+  }
```

Comment:
```suggestion
  if (minusGravity.isNan())
    minusGravity = WbVector3(0.0, 1.0, 0.0);
```
https://github.com/cyberbotics/webots/wiki/CPP-Coding-Style#cs360-single-if-while-or-for-statement-must-be-written-without-brackets-the-statement-part-should-be-put-on-a-separate-line-in-case-of-nested-block-keep-brackets

Reply:
True, I should set it in my auto-indent plugin...